test/NVQPP/qir_test_cond_for_break.cpp

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// clang-format off
+// RUN: nvq++ --target quantinuum --emulate %s -o %basename_t.x && ./%basename_t.x | FileCheck %s
+// XFAIL: *
+// ^^^^^ Produces error: 'cc.loop' op not a simple counted loop
```

Comment:
See #768

Reply:
I documented the issue in #781 so that we can merge this PR.